src/pool/srv_pool.c

```diff
@@ -2040,14 +2104,35 @@ read_db_for_stepping_up(struct pool_svc *svc, struct pool_buf **map_buf_out,
 			DP_UUID(svc->ps_uuid));
 	}
 
+	prop_entry = daos_prop_entry_get(prop, DAOS_PROP_PO_GLOBAL_VERSION);
+	D_ASSERT(prop_entry != NULL);
+	arg.global_ver = prop_entry->dpe_val;
+	prop_entry     = daos_prop_entry_get(prop, DAOS_PROP_PO_OBJ_VERSION);
+	D_ASSERT(prop_entry != NULL);
+	arg.obj_ver = prop_entry->dpe_val;
+	arg.iv_hdls = NULL;
+	rc = rdb_tx_iterate(&tx, &svc->ps_handles, false /* backward */, add_conn_cb, &arg);
+	if (rc != 0) {
+		D_ERROR("Failed to find hdls for evict pool " DF_UUIDF " connections: " DF_RC "\n",
```

Comment:
[Nit] Should be `DL_ERROR(rc, DF_UUID ": failed to ...`.

Reply:
sure.